microceph/ceph/config.go

```diff
@@ -93,20 +104,24 @@ type ConfigDumpItem struct {
 }
 type ConfigDump []ConfigDumpItem
 
-func SetConfigItem(c types.Config) error {
-	configTable := GetConstConfigTable()
+func SetConfigItem(c types.Config, bypassChecks bool) error {
+	// safety checks
+	canSet, err := canSetConfig(c.Key)
+	if !canSet && !bypassChecks {
+		return err
```

Comment:
I think adding some context to the error would be good here, sth like

```
return fmt.Errorf("not allowed to set config item ...")
```

Reply:
ack, the internal error is also descriptive so I have stacked them.